src/wallet/rpcwallet.cpp

```diff
@@ -2581,6 +2584,212 @@ UniValue fundrawtransaction(const JSONRPCRequest& request)
     return result;
 }
 
+UniValue bumpfee(const JSONRPCRequest& request)
+{
+    if (!EnsureWalletIsAvailable(request.fHelp))
+        return NullUniValue;
+
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 2)
+        throw runtime_error(
+                            "bumpfee \"txid\" ( options ) \n"
+                            "\nBumps the fee of an opt-in-RBF transaction T, replacing it with a new transaction B.\n"
+                            "An opt-in RBF transaction with the given txid must be in the wallet.\n"
+                            "The command will not add new inputs or alter existing inputs.\n"
```

Comment:
Thanks, I changed the wording to describe the current behavior but suggest it could be improved in the future.